.travis.yml

```diff
@@ -1,37 +1,50 @@
 language: java
-sudo: false
-jdk:
-- openjdk8
-- openjdk9
-- openjdk10
-- openjdk11
+
 cache:
   directories:
-  - ".autoconf"
-  - "$HOME/.m2"
+    - ".autoconf"
+    - "$HOME/.m2"
+
+before_install:
+  - wget https://github.com/sormuras/bach/raw/master/install-jdk.sh
```

Comment:
It seems to be included already:
https://docs.travis-ci.com/user/languages/java/#switching-jdks-to-java-10-and-up-within-one-job